ts/src/test/static/request/gate.json

```diff
@@ -2,6 +2,9 @@
     "exchange": "gate",
     "skipKeys": ["text"],
     "outputType": "json",
+    "options": {
```

Comment:
@Dan-krm That happened because the test was stopping at the `isUnifiedAccountEnabled` call because you were not providing a default value for options in tests.